index.html

```diff
@@ -1,14 +1,164 @@
+        </div>
+        <div class="img2wrapper">
+          <img class="img2" src="file:///Users/jaysondurante/Desktop/Jayson/Burlington%20Code%20Academy/Remock%20Proj%20Pics/grass.jpg"/>
+          <div>IMAGE TWO</div>
+        </div>
+        <div class="img3wrapper">
+          <img class="img3" src="file:///Users/jaysondurante/Desktop/Jayson/Burlington%20Code%20Academy/Remock%20Proj%20Pics/glacier_water.jpg"/>
+          <div>IMAGE THREE
+          </div>
+        </div>
+      </div>
+    <br>
+      <div class="button">
+        <button class="button1" type="button"><a target="_blank" href="blog-post.html">Tell Me More!</a></button>
+      </div>
+    <br>
+      <div class="quote">
+        <div class="quote-text">
+        <h3>Use this space for a great quote.</h3>
+        </div>
+      </div>
+
+      <!-- Main Content Page 1-->
+
+      <div class="main-content1-wrapper">
+        <div class="main-content1">
+          <h4 class="main-content1-text1">I DON'T WANT TO SAY <b>IT'S THE ALIENS...</b><br>BUT IT'S THE ALIENS.</h4>
+          <img class="main-content1-img1" src="file:///Users/jaysondurante/Desktop/Jayson/Burlington%20Code%20Academy/Remock%20Proj%20Pics/satellite-aliens.jpg"/>
+          <h3>TOTALLY...</h3>
+          <p class="img1-desc1">Lorem ipsum dolor sit amet consectetur, adipisicing elit. Ipsa eveniet atque placeat neque quasi vel, facilis ullam nemo veritatis, amet, exercitationem officiis consectetur totam. Deleniti porro ipsa optio obcaecati minima placeat saepe officiis accusamus animi amet, voluptates dolorem perspiciatis, culpa earum repellat facere. Iusto, accusantium velit labore nesciunt ea ullam.</p>
+          <button class="button2" type="button"><a target="_blank" href="continue-reading.html">CONTINUE READING</a></button>
+          <br>
+          <p class="underline">                                                            </p>
+          <h4 class="main-content1-text1">AWESOME DEMO PHOTO</h4>
+          <img class="main-content1-img1" src="https://picsum.photos/300/203"/>
+          <p class="img1-desc1">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eos in sint, minus qui eius aliquid id molestiae sunt</p>
+          <p>reiciendis atque? Cupiditate nam minima provident aut fugit tempora recusandae ab, dolor unde labore soluta sint reiciendis velit, eum molestias aspernatur praesentium, fuga dolorem non fugiat perferendis nesciunt. Porro nesciunt ex cum unde exercitationem corporis expedita deleniti esse? Explicabo a amet eligendi at non molestias facilis autem doloremque. Quos dolore placeat illum? Sunt culpa perferendis quas distinctio</p> 
+          <p>fugit veniam tenetur est illum corrupti, ipsum accusamus doloremque laborum accusantium aut hic consequuntur delectus! Officiis repellat distinctio voluptas nostrum unde eveniet cupiditate non eligendi.</p>
+          <button class="button2" type="button"><a target="_blank" href="continue-reading.html">CONTINUE READING</a></button>

```

Comment:
Using more descriptive class names would benefit any future work being done on this code as well. You want to make them as clear as your variable names when writing JS